src/Components/Header/Header.jsx

```diff
@@ -2,12 +2,15 @@ import React from 'react';
 import { useList } from '../../contexts/ListContext';
 
 export default function Header() {
-  const { items } = useList();
+  const { items, handleClearItems } = useList();
 
   return (
     <>
       <header>Shopping List</header>
       <span>Items: {items.length}</span>
+      <button aria-label="clear-button" onClick={handleClearItems}>
```

Comment:
For screen readers, buttons don't need aria-labels. The reader will tell the user it's a button and will also read whatever text is between the <button> tags. So this one might read "Clear Items button"
However, if you have a bunch of buttons of the same name on the screen, you might need to aria-label it with a temperate literal name with unique identifier for testing purposes.